rpm/ngfd-plugin-droid-vibrator.spec

```diff
@@ -45,3 +45,4 @@ fi
 %defattr(-,root,root,-)
 %doc README COPYING
```

Comment:
We should also move everything else than COPYING to devel package or just leave out.